tests/unit/handlers/L1RedemptionHandlerFactory.test.ts

```
@@ -0,0 +1,651 @@
+
+      for (let i = 0; i < iterations; i++) {
+        const config = {
+          ...mockEvmConfig,
+          chainName: `TestChain${i}`,
+        };
+        handlers.push(L1RedemptionHandlerFactory.createHandler(config));
+      }
+
+      expect(handlers).toHaveLength(iterations);
+      expect(L1RedemptionHandler).toHaveBeenCalledTimes(iterations);
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)